jupyter/pytorch/ubi9-python-3.11/Pipfile

```diff
@@ -41,14 +41,14 @@ mysql-connector-python = "~=9.3.0"
 
 odh-elyra = "==4.2.1"
 
-jupyterlab = "==4.2.7"
+jupyterlab = "==4.4.4"
```

Comment:
@coderabbitai this update will require thorough support by automated tests; can you please suggest additions to our `tests/` directory where we keep automated tests to build a comprehensive test strategy for this and future jupyterlab updates?